packages/astro/components/Picture.astro

```diff
@@ -0,0 +1,57 @@
+---
+import { getImage, type LocalImageProps, type RemoteImageProps } from 'astro:assets';
+import type { GetImageResult, ImageOutputFormat } from '../dist/@types/astro';
+import { isESMImportedImage } from '../dist/assets/internal';
+import { AstroError, AstroErrorData } from '../dist/core/errors/index.js';
+import type { HTMLAttributes } from '../types';
+
+type Props = (LocalImageProps | RemoteImageProps) & {
+	formats?: ImageOutputFormat[];
```

Comment:
Is `svg` ok here?

Reply:
Yes! It's okay for it to be used because the input might be a `svg`. Additionally, we don't support it ourselves (and our services does successfully prevents it), but an image service could implement a raster to vector function and fully support using a `svg` here no matter the input format